pkgs/by-name/xb/xbar/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenvNoCC, fetchurl, undmg, ... }:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "xbar";
+  version = "2.1.7-beta";
+  src = fetchurl {
```

```suggestion
  version = "2.1.7-beta";
  
  src = fetchFromGitHub {
```
just some formatting, and if you go the way of building from source (which you should) then you can switch to the fetchFromGitHub wrapper

---

Done